nifi-registry/nifi-registry-docker-maven/dockermaven/Dockerfile

```diff
@@ -34,7 +38,7 @@ RUN unzip ${NIFI_REGISTRY_BASE_DIR}/nifi-registry-${NIFI_REGISTRY_VERSION}-bin.z
     && ln -s ${NIFI_REGISTRY_HOME} ${NIFI_REGISTRY_BASE_DIR}/nifi-registry-${NIFI_REGISTRY_VERSION}
 
 
-FROM openjdk:8-jre
+FROM eclipse-temurin:11-jdk
```

Comment:
Should/could we be using the `jre` instead of `jdk` for the `nifi-registry` images?

Reply:
I'll look into this. I think nifi-registry used to require JDK for some Flyway migration scripts that use java syntax and need runtime compilation or something like that, but it might be fixed. Do you recall @bbende?

Reply:
Since the current image is `jre`, I agree that we should stay with the `jre` version.

Reply:
I think the issue is resolved...

In 0.1.0 release of registry it was because of this:

https://github.com/apache/nifi-registry/blob/rel/nifi-registry-0.1.0/nifi-registry-framework/src/main/resources/db/migration/V1.3__DropBucketItemNameUniqueness.sql

In 0.2.0 we re-did the schema and removed that and we implemented an automatic migration from old DB to new, so the requirement was anyone on 0.1.0 had to first migrate to 0.2.0 before any further upgrades.

At this point the original schema has been completely removed.

Reply:
Awesome, thanks for confirming @bbende!

Reply:
Updated in latest commit